tests/test_standard_tags.py

```diff
@@ -79,6 +79,7 @@ def setup_url_with_sensitive_query_string_legacy(self):
     # when tracer is updated, add (for example)
     @irrelevant(context.library >= "java@1.21.0", reason="java released the new version at 1.21.0")
     @irrelevant(context.library >= "python@1.18.0rc1", reason="python released the new version at 1.19.0")
+    @irrelevant(context.library >= "dotnet@2.40", reason="dotnet released the new version at 2.39.0")
```

Comment:
Be careful to strict (or not) equality, based on your comment, it's : 

```suggestion
    @irrelevant(context.library >= "dotnet@2.39", reason="dotnet released the new version at 2.39.0")
```

Reply:
same as below, we just had a release to 2.40 actually so it's 2.40 now.
And even then, we dont wanna play for 2.40 as the PR is not merged yet.